matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/settings/DefaultLightweightSettingsStorage.kt

```diff
@@ -28,21 +29,20 @@ import javax.inject.Inject
  * on the sdk without using the database. This should be used just for sdk/user preferences and
  * not for large data sets
  */
-
-class LightweightSettingsStorage  @Inject constructor(
+internal class DefaultLightweightSettingsStorage @Inject constructor(
```

Comment:
not for this PR but I wonder if these settings should be client side rather than in the SDK

Reply:
They are meant to be SDK side, client side we have vectorPreferences. You can't read `vectorPreferences` from SDK side. It's mainly for dynamic settings changes, MatrixConfiguration cannot be changed dynamically.

Reply:
The main problem to me, is that those settings are SDK scope and should be session scope.